README.md

```diff
@@ -29,17 +29,32 @@ marks](CaliptraTrademarkPolicy.md).
 * [FMC](https://github.com/chipsalliance/caliptra-sw/tree/main/fmc#readme)
 * [Firmware](https://github.com/chipsalliance/caliptra-sw/blob/main/runtime/README.md)
 
+## Security policy
+
+For information on the Caliptra security policy, how to report a vulerability,
+and published security advisories, refer to
+[Caliptra security policy](https://github.com/chipsalliance/caliptra/security/policy).
+
+### Product creator registration
+
+Companies or individuals building products that integrate Caliptra are
+encouraged to register as a Caliptra product creator. Product creators will
```

Comment:
singular-->plural

`encouraged to register as a Caliptra product creator`-->`encouraged to register as Caliptra product creators`

Reply:
done